linguee/__init__.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+
+"""linguee extension
+
+translate ger-eng with linguee
+
+Synopsis: <trigger> <word>"""
+
+
+from albertv0 import *
+import requests
+import bs4
+import os
+
+
+lang = "deutsch-englisch"
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Linguee-deutsch-englisch"
+__version__ = "0.1"
+__trigger__ = "lin "
+__author__ = "Lucky Lukert, David Koch"
+__dependencies__ = ["beautifulsoup4"]
```

Comment:
i know this comes in handy, but it is annoying for end users and can be avoided (unless I missed something)

Reply:
ok I'll try to remove this dependency